### ISSUE TITLE
Refactor all styling to fix contrast

### ISSUE THREAD

**Issue by EstiShay** [EstiShay/estishay.github.io#8]
Default link color from imported styles does not meet contrast requirements. Will have to completely rework styling to remove the default stylesheet I have been relying on.
